tools/pypi/generate-pypi-mfa-qr.py

```diff
@@ -0,0 +1,35 @@
+import qrcode
+import subprocess
+from io import BytesIO
+from PIL import Image
+from azure.identity import DefaultAzureCredential
+from azure.keyvault.secrets import SecretClient
+
+vault_url = "https://mmlspark-keys.vault.azure.net"
+secret_name = "pypi-mfa-uri"
+
+def retrieve_secret_from_keyvault(vault_url, secret_name):
+    try:
+        return SecretClient(vault_url=vault_url, credential=DefaultAzureCredential()).get_secret(secret_name).value
+    except Exception as e:
+        print(f"Error: Failed to retrieve the secret from Azure Key Vault. {e}")
```

Comment:
nit: dont catch errors if it truly is an error, instread rethrow with a better error message on top